src/modules/db_operations/operations.py

```diff
@@ -132,6 +135,23 @@ def drop_and_populate_phenotype_metadata(app, db, species, reload_files=True):
   logger.info("Loading phenotypes...")
   etl_manager.load_tables(PhenotypeMetadata, species_list=species)
 
+def drop_and_populate_phenotypes(app, db, species, reload_files=True):
```

Comment:
we should add docstring to the function args and be explicit about inputs and side-effects (dropped tables, etc).